collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadListActivity.java

```diff
@@ -577,7 +577,7 @@ public void formListDownloadingComplete(HashMap<String, ServerFormDetails> formL
         } else {
             switch (exception.getType()) {
                 case FETCH_ERROR:
-                case UNKNOWN_HOST:
+                case UNREACHABLE:
```

Comment:
Really we're using this to flag when we can't get to the server so the naming felt like it should be more general.